roles/zabbix_proxy/tasks/Debian.yml

```diff
@@ -129,108 +94,3 @@
   become: true
   tags:
     - install
-
-- name: "Debian | Installing zabbix-proxy-{{ zabbix_proxy_database }}"
-  ansible.builtin.apt:
-    pkg: "zabbix-proxy-{{ zabbix_proxy_database }}"
-    update_cache: true
-    cache_valid_time: 0
-    force: true
-    state: "{{ zabbix_proxy_package_state }}"
-    default_release: "{{ ansible_distribution_release }}"
-  environment:
-    http_proxy: "{{ zabbix_http_proxy | default(None) | default(omit) }}"
-    https_proxy: "{{ zabbix_https_proxy | default(None) | default(omit) }}"
-  register: is_zabbix_proxy_package_installed
-  until: is_zabbix_proxy_package_installed is succeeded
-  become: true
-  tags:
-    - install
-
-- name: "Debian | Installing zabbix-sql-scripts"
-  ansible.builtin.apt:
-    pkg: zabbix-sql-scripts
-    state: "{{ zabbix_proxy_package_state }}"
-    update_cache: true
-    cache_valid_time: 0
-    default_release: "{{ ansible_distribution_release }}"
-  environment:
-    http_proxy: "{{ zabbix_http_proxy | default(None) | default(omit) }}"
-    https_proxy: "{{ zabbix_https_proxy | default(None) | default(omit) }}"
-  register: zabbix_proxy_package_sql_installed
-  until: zabbix_proxy_package_sql_installed is succeeded
-  when:
-    - zabbix_proxy_version is version('6.0', '>=')
-  become: true
-  tags:
-    - install
-
-- name: "Debian | Install Ansible module dependencies"
-  ansible.builtin.apt:
-    name: "{{ zabbix_python_prefix }}-psycopg2"
-    state: present
-  environment:
-    http_proxy: "{{ zabbix_http_proxy | default(None) | default(omit) }}"
-    https_proxy: "{{ zabbix_https_proxy | default(None) | default(omit) }}"
-  register: zabbix_proxy_dependencies_installed
-  until: zabbix_proxy_dependencies_installed is succeeded
-  become: true
-  when:
-    - zabbix_proxy_database_creation
-  tags:
-    - install
-    - dependencies
-
-- name: "Debian | Install Mysql Client package"
```

Comment:
I actually think that doesn't matter. They're not conflicting packages, in that they can both be present at the same time on debian systems. Infact they were both present on the zabbix_server tests, thanks to [prepare.yml](https://github.com/ansible-collections/community.zabbix/blob/main/molecule/zabbix_server/prepare.yml#L113-L119)

So what's gonna happen is that python3-pymysql will get installed (if it's not already present), and the community.mysql collection will use pymysql  instead of mysqldb when the playbook gets to that part.

Reply:
OK fair point.